src/core/core.controller.js

```diff
@@ -985,8 +1014,12 @@ helpers.extend(Chart.prototype, /** @lends Chart */ {
 		// Find Active Elements for hover and tooltips
 		if (e.type === 'mouseout') {
 			me.active = [];
+			me._lastEvent = null;
 		} else {
 			me.active = me.getElementsAtEventForMode(e, hoverOptions.mode, hoverOptions);
+			if (me._lastEvent !== 'click') {
```

Comment:
Why are clicks special here?

Reply:
That is because click events on legends and chart elements also pass here and you don't want to replay those events when updating a chart. Then you get behaviour as mentioned in @nagix his previous review.

Reply:
```js
if (e.type !== 'click') {
````
(me._lastEvent is never 'click', its the previous event object)